terraform/application/config/qa.tfvars.json

```diff
@@ -0,0 +1,6 @@
+{
+    "cluster": "test",
+    "namespace": "bat-qa",
+    "deploy_azure_backing_services": false,
```

Comment:
deploy_azure_backing_services should be true for qa (or removed as this should be the default)